### PR TITLE
`Filter by Attribute` block settings design update

### PR DESCRIPTION
**Pull request by albarin**
This PR updates the design of the `Filter by Attribute` block settings according to the new design.
**It only updates the block settings panel, not the block rendering.**

Related to https://github.com/woocommerce/woocommerce-blocks/issues/6843

#### User Facing Testing

1. Add the `Filter by Attribute` block to a page.
2. Check the copy and the design matches with the new design.
3. Check no regression has happened.

### WooCommerce Visibility
* [ ] WooCommerce Core
* [x] Feature plugin
* [ ] Experimental

### Changelog

> Update the design of the Filter by Attribute block settings panel.

### CODE REVIEW

File: assets/js/blocks/attribute-filter/edit.tsx [hunk 1]
```diff
@@ -309,7 +308,7 @@ const Edit = ( { attributes, setAttributes, debouncedSpeak }: EditProps ) => {
 					</ToggleGroupControl>
 					<ToggleControl
 						label={ __(
-							'Filter button',
+							"Show 'Apply filters' button",
```
Comment (dinhtungdu) [inline assets/js/blocks/attribute-filter/edit.tsx:311]: I notice that this label is different between blocks in the design:
- `'Apply filters' button`: the Stock filter and Attribute filter blocks.
- `Show 'Apply filters' button`: the Price filter and Rating filter blocks.

If we haven't, I think we should confirm with @vivialice to use only one label across filter blocks.

**Review by dinhtungdu — COMMENTED** [review]
LGTM! I just left a minor comment about the filter button toggle label.

File: assets/js/blocks/attribute-filter/edit.tsx (hunk 1 above)
Comment (tjcafferkey) [reply assets/js/blocks/attribute-filter/edit.tsx:311]: Yep, I used "Show 'Apply filters' button" on _Filter by Stock_ component despite it saying "'Apply filters' button" as its consistent with your Filter by Price component which has already been merged, and its a more descriptive label. @vivialice would you agree?

Comment (vivialice) [reply assets/js/blocks/attribute-filter/edit.tsx:311]: @tjcafferkey Yes I agree, thank you.

**Review by dinhtungdu — APPROVED** [review]
🎉

**Pull request merged**